python/src/worker/tasks/__init__.py

```diff
@@ -6,6 +6,8 @@ class Task(ABC):
 
     def __init__(self, msg):
         self.task_def = msg["body"]
+        self.request = msg
```

Comment:
looks like this `self.request` is only used in the cluster cells task. If that is indeed the case, wouldn't it be better to move this `self.request` to be a variable only for the `ClusterCells` class and add it in its init method?

Reply:
hmm, i seem to recall there was an issue with it but it seems to be working. i'll put it back there and see